.eslintrc.js

```diff
@@ -257,6 +257,11 @@ module.exports = {
         './packages/eslint-plugin/src/rules/**/*.ts',
       ],
       rules: {
+        'eslint-plugin/require-meta-docs-description': [
+          'error',
+          { pattern: '^(Enforce|Require|Disallow) .+[^. ]$' },
```

Comment:
Nice, TIL about this one. Eventually maybe we'll even narow this list down to two 😄